templates/agent-conf.d/http_check.yaml.erb

```diff
@@ -28,7 +28,9 @@ instances:
 <% if instance['collect_response_time'] -%>
     collect_response_time: <%= instance['collect_response_time'] %>
 <% end -%>
-    disable_ssl_validation: <%= instance['disable_ssl_validation'] %>
+<% if instance['disable_ssl_validation'] -%>
```

Comment:
I _think_ this will fail if the key doesn't exist... can you try with `has_key(instance, 'disable_ssl_validation')` instead?